beta/vercel.json

```
@@ -254,5 +254,16 @@
       "destination": "https://5a046bf5a6188f4b8fa4938a--reactjs.netlify.com",
       "permanent": false
     }
+  ],
+  "headers": [
+    {
+      "source": "/fonts(.*)",
```

```suggestion
      "source": "/fonts/.*",
```
Also we might want to check if this works or we need to do `\/fonts\/.*` cc: @gaearon